deploy/.env.example

```diff
@@ -11,9 +11,16 @@ BRIDGEABLE_TOKEN_SYMBOL=TEST
 BRIDGEABLE_TOKEN_DECIMALS=18
 
 HOME_RPC_URL=https://sokol.poa.network
-HOME_BRIDGE_OWNER=0x
-HOME_VALIDATORS_OWNER=0x
-HOME_UPGRADEABLE_ADMIN=0x
+# HOME_OWNER_MULTISIG=0x
+HOME_BRIDGE_OWNER=0x ##
+HOME_VALIDATORS_OWNER=0x ##
+HOME_FACTORY_OWNER=0x
+HOME_MAPPER_OWNER=0x
+HOME_UPGRADEABLE_ADMIN=0x ##
+# HOME_UPGRADEABLE_ADMIN_VALIDATORS=0x
```

Comment:
Please correct me if I'm wrong, but I think the following variables are not being used and can be removed:
```
# HOME_OWNER_MULTISIG=0x
# FOREIGN_OWNER_MULTISIG=0x
# HOME_UPGRADEABLE_ADMIN_VALIDATORS=0x
# HOME_UPGRADEABLE_ADMIN_BRIDGE=0x
# HOME_UPGRADEABLE_ADMIN_FACTORY=0x
# HOME_UPGRADEABLE_ADMIN_MAPPER=0x
# FOREIGN_UPGRADEABLE_ADMIN_VALIDATORS=0x
# FOREIGN_UPGRADEABLE_ADMIN_BRIDGE=0x
# FOREIGN_UPGRADEABLE_ADMIN_FACTORY=0x
```

Also I think it would be good if we include the following variables that are used on deploy scripts:
```
HOME_BRIDGE_VALIDATORS_IMPLEMENTATION_ADDRESS=
HOME_BRIDGE_IMPLEMENTATION_ADDRESS=
FOREIGN_BRIDGE_VALIDATORS_IMPLEMENTATION_ADDRESS=
FOREIGN_BRIDGE_IMPLEMENTATION_ADDRESS=
```

Reply:
You are correct, this variable are obsolete, I will remove them
Also, I will add the optional variables to the `.env` templates as you suggested.